akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -31,6 +30,9 @@ final case class DeleteMessagesSuccess(toSequenceNr: Long)
  */
 final case class DeleteMessagesFailure(cause: Throwable, toSequenceNr: Long)
 
+@SerialVersionUID(1L)
+case object RefusedToStash
```

Comment:
this would require Java api also, but I don't think we should provide a default class for this at all, better that the application defines the message